enterprise/frontend/src/embedding-sdk/types/theme/index.ts

```diff
@@ -36,6 +36,12 @@ export interface MetabaseColors {
   /** Primary brand color used for buttons and links */
   brand?: string;
 
+  /** Lighter variation of the brand color. Used for hover and accented elements. */
+  "brand-hover"?: string;
+
+  /** Lightest variation of the brand color. Used for hover and accented elements. */
+  "brand-hover-light"?: string;
```

Comment:
We already have `background-hover` that maps to `bg-light`, so it make sense to also do this for brand colors.

Reply:
Is it that some hover element use `brand-light` while others use `brand-lighter`?

It feels to me there should only be one hover color.

This can be clean up later on of course.

Reply:
@WiNloSt yes, for some reason we have both `brand-light` and `brand-lighter` for hover colors in Metabase 😨